Octokit.Tests/Models/IssueTest.cs

```diff
@@ -257,7 +257,7 @@ public void CreatesAnIssueUpdateRequestObject()
 
             var update = issue.ToUpdate();
 
-            Assert.Null(update.Labels);
+            Assert.NotNull(update.Labels);
```

Comment:
Ok cool. Let me dig into this further as `ToUpdate` shouldn't create the `Labels` array by default unless it already exists on the issue. There's some [assumptions](https://developer.github.com/v3/issues/#edit-an-issue) about what we `POST` as part of editing an issue that may have changed (emphasis mine):

> Labels to associate with this issue. Pass one or more Labels to replace the set of Labels on this Issue. **Send an empty array (`[ ]`) to clear all Labels from the Issue.** NOTE: Only users with push access can set labels for issues. Labels are silently dropped otherwise.